packages/primitives/button/test/index.ts

```diff
@@ -0,0 +1,56 @@
+import test from 'tape'
+import { getAppPage } from 'foreal'
+
+type TFixtures = 'bare-bones' | 'disabled'
+const getFixture = (filename: TFixtures) => `./fixtures/${filename}.tsx`
+
+const PACKAGE = '@primitives/button'
+
+test(`${PACKAGE} has proper role`, async (t) => {
```

Comment:
it looks like each test looks like:
```
const entryPointPath = require.resolve(getFixture('bare-bones'))

  const page = await getAppPage({
    entryPointPath,
  })
```
....
...
..
```
await page.close()
```
do you think we should make some global helping function for it?

Reply:
I totally agree with the idea. But I would prefer doing it on an isolated PR, since it's going to be a method which needs to be exposed to all packages it's nice to keep this addition as granular as possible.

@DonGideon do you agree? I'd happily get to work on it right now if so...

Reply:
sure we can wait with it 👍